lib/content-services/src/lib/directives/node-lock.directive.ts

```diff
@@ -33,7 +33,7 @@ export class NodeLockDirective implements AfterViewInit {
 
     @HostListener('click', [ '$event' ])
     onClick(event) {
-        event.preventDefault();
+        event.stopPropagation();
```

Comment:
Are you sure not both methods need to be called?

Reply:
preventDefault is for links or buttons in forms to stop browsers default behaviour as far as I know, doesn't have any benefit  here